### PR TITLE
Backpacks

### PR DESCRIPTION
**Pull request by 21Melkuu**
## About the PR

Add new explosion-proof backpack in aplink for 2 tk
## Why / Balance

Mobility for nuke ops is an important part. At the moment the ops cannot take explosives with him, and medicine with him in a regular gray  backpack due to the lack of explosion protection. To solve this problem and add a little variability I propose this backpack

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: 21Melkuu
- add: Add new explosion-proof backpack in aplink.

### CODE REVIEW

**Comment by Dutch-VanDerLinde** [comment]
i think this should go in the wearables category rather than pointless, its not exactly pointless

**Comment by Dutch-VanDerLinde** [comment]
also instead of an uplink purchase, what about adding it to the SyndieDrobe since the nukies get that on the planet

**Comment by 21Melkuu** [comment]
> also instead of an uplink purchase, what about adding it to the SyndieDrobe since the nukies get that on the planet

I still think that a backpack like this has its advantages, so it should be price.

**Comment by 21Melkuu** [comment]
> i think this should go in the wearables category rather than pointless, its not exactly pointless

you're probably right

**Comment by kerisargit** [comment]
Please, I beg you, Satchel too. It’s just beautiful on the medic, you can lay everything out neatly

**Comment by 21Melkuu** [comment]
> Please, I beg you, Satchel too. It’s just beautiful on the medic, you can lay everything out neatly

if approves this PR I make satchel

**Comment by Golinth** [comment]
> Please, I beg you, Satchel too. It’s just beautiful on the medic, you can lay everything out neatly

I take the satchel from the clothesmate anyway, so please god add the satchel too.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -1287,6 +1287,16 @@
...
 
+- type: listing
+  id: UplinkBackpackSyndicate
+  name: uplink-backpack-syndicate-name
+  description: uplink-backpack-syndicate-desc
+  productEntity: ClothingBackpackSyndicate
+  cost:
+    Telecrystal: 2
+  categories:
+    - UplinkWearables
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1298]:
```suggestion
  categories:
    - UplinkWearables
```

Please use an IDE so your yaml gets trimmed properly.

File: Resources/Prototypes/Entities/Clothing/Back/backpacks.yml [hunk 2]
```diff
@@ -248,6 +248,18 @@
   - type: Sprite
     sprite: Clothing/Back/Backpacks/ertclown.rsi
 
+#Syndicate
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackSyndicate
+  name: syndicate backpack
+  description:
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Back/backpacks.yml:256]:
```suggestion
  description:
```

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
Please make your PR names descriptive in future.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**